packages/cli/src/validate-scan-arguments.ts

```diff
@@ -24,4 +24,15 @@ export function validateScanArguments(args: ScanArguments): void {
     if (args.updateBaseline === true && isEmpty(args.baselineFile)) {
         throw new Error('Option --updateBaseline requires option --baselineFile.');
     }
+
+    if (!args.crawl && (!isEmpty(args.serviceAccountName) || !isEmpty(args.serviceAccountPass))) {
+        throw new Error('Options --serviceAccountName and --serviceAccountPass are only supported with --crawl.');
```

Comment:
do you mean `supported with --crawl option`?
update to `serviceAccountPassword`

Reply:
I was matching this existing error message: 
```
    if (!args.crawl && !isEmpty(args.baselineFile)) {
        throw new Error('Option --baselineFile is only supported with --crawl.');
    }
```

Should I change both or leave it?

Reply:
we can update baselineFile message as well to match